diagnostic.md

```diff
@@ -14,7 +17,7 @@ list items (`li`) using jQuery? This is intentionally vague. (Assume starting
 at the `OL` DOM node).
 
 <!-- your answer starts here -->
-
+$("li").val()
```

Comment:
`$('li');` will select all `<li>` elements, but invoking `.val()` on the return value will result in `undefined`.